.github/release.yml

```diff
@@ -0,0 +1,6 @@
+changelog:
+  exclude:
+    authors:
+      - dependabot # Automated dependency updates
+      # - slicerbot # Handles Slicer.crt certificate updates (see update-slicer-certificate-bundle.yml)
```

Comment:
```suggestion
      # - slicerbot # Handles Slicer.crt certificate updates (see update-slicer-certificate-bundle.yml). These updates are not excluded from the changelog, because they update the Slicer.crt certificate, and it is useful to document these changes.
```